### ISSUE TITLE
Error with consistency style rule?

### ISSUE THREAD

**Issue by ChrisChinchilla** [vale-cli/vale#129]
I have been testing creating a new consistency check, and couldn't get it to work, so thought I'd just try the example rule from the [docs](https://errata-ai.github.io/vale/styles/#consistency).

But that doesn't trigger either. Here's my test markdown file:

```
# Test

The centre issue is that everything is going to the center.
```

Is the example config wrong, or something else? Latest vale, installed with Homebrew, running on a mac. Tried with VSC package and command line.

**Comment by jdkato** [comment]
Using that snippet with the files below and `v1.4.1`, I get:

```console
 test.md
 3:53  error  Inconsistent spelling of        Test.Spelling 
              'center'                                      

✖ 1 error, 0 warnings and 0 suggestions in 1 file.                              
```

Can you reproduce this?

---

```ini
# .vale.ini
StylesPath = styles
MinAlertLevel = suggestion

[*.md]
BasedOnStyles = Test
```

```yaml
# styles/Test/Spelling.yml
extends: consistency
message: "Inconsistent spelling of '%s'"
level: error
scope: text
ignorecase: true
nonword: false
# We only want one of these to appear.
either:
  advisor: adviser
  centre: center
```

**Comment by ChrisChinchilla** [comment]
Huh, now that's strange. For ages I've had that `BasedOnStyles` line commented out and everything worked anyway (about a dozen styles), but now for this particular style, it only works with that line uncommented. I guess I need to rejig a lot of my ini files to re-add that line. Did something change recently in a version and I just didn't notice until now?

Because even with that line uncommented, I only have one set included, but other styles not included still work too.

So not sure if this is another bug, or working as designed.

**Comment by jdkato** [comment]
Nothing style-related has changed since v1.0.0: 

- You can activate an entire style by listing it in `BasedOnStyles`; or
- you can individually activate rules from various styles (without using `BasedOnStyles`) with statements like `SomeStyle.SomeRule = warning`.

I suspect the latter case explains how some of your styles are working without being listed in `BasedOnStyles`. If you don't think that's the case, I'd like to see your INI file to investigate further.

**Comment by ChrisChinchilla** [comment]
OK, but then this kind of brings me back to the start of this issue. 

```
StylesPath = vale-styles
MinAlertLevel = suggestion # suggestion, warning or error

[*.{md,txt,rst,adoc}]
ttd.QuoteType = warning
```

should trigger that check? But it doesn't, only if I use  `BasedOnStyles` to activate the whole set.  From my testing, it only seems to be styles based on `consistency`. If I then change the INI file to:

```
StylesPath = vale-styles
MinAlertLevel = suggestion # suggestion, warning or error

[*.{md,txt,rst,adoc}]
ttd.QuoteType = warning
ttd.Spelling = warning
```

Then the spelling check activates, but not the other. So… Hmm.

**Comment by jdkato** [comment]
I think I was able to figure this out and it *should* be fixed in [`v1.4.2`](https://github.com/errata-ai/vale/releases/tag/v1.4.2).

**Comment by ChrisChinchilla** [comment]
@jdkato That fixed it! Now I just need to figure out the style I'm creating, which I'll bug you about separately ;)
